### ISSUE TITLE
Fix FP S6747 (`no-unknown-property`): Don't raise for `css` attribute

### ISSUE THREAD

**Issue by ilia-kebets-sonarsource** [SonarSource/SonarJS#4632]
The `css` attribute is used in [FlowJS](https://github.com/flowjs/flow.js) and [Emotion](https://emotion.sh/docs/introduction).

Reproducer:

```js
const Foo = () => {
    const things = [
      {
        label: 'foo',
        value: 'bar',
      },
    ];

    return (
      <>
        {things.map((thing) => {
          return (
            thing.value && (
              <div css={style.flexCell}>
                <label css={styles.textLabel}>{thing.label}</label>
                <span css={styles.textValue}>{thing.value}</span>
              </div>
            )
          );
        })}
      </>
    );
};
```

**Comment by ilia-kebets-sonarsource** [comment]
closing as I incorporated this issue into https://github.com/SonarSource/SonarJS/issues/4633
